code/game/objects/items/maintenance_loot.dm

```diff
@@ -0,0 +1,43 @@
+//The items defined in this file are intended be scarce maintenance loot items some of these items are used as a non-renewable resource in crafting or ghetto chem.
+//Exercise good judgement and don't add these to a lathe willy nilly.
+
+//Saw-tier bulky & blunt weapon. A decent bone breaker. Source of lead reagent.
+//Add lead material to this once implemented.
+/obj/item/lead_pipe
+	name = "lead pipe"
+	icon = 'icons/obj/maintenance_loot.dmi'
+	icon_state = "lead_pipe"
+	inhand_icon_state = "lead_pipe"
+	lefthand_file = 'icons/mob/inhands/weapons/melee_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/melee_righthand.dmi'
+	//wow, lore
+	desc = "A hefty lead pipe.\nLead in an uncommon sight in this sector after being phased out due to employee health concerns.\nThose of a more cynical disposition assume that the NT lead ban is a scheme to prevent the metal from being diverted to Syndicate ammunition factories."
```

Comment:
i'd prefer if the long description had a ` \` (space and a backslash) and then went to the next line so you don't have to scroll to the right to read it